source/Cosmos.HAL2/Audio/SampleFormat.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+
+namespace Cosmos.HAL.Audio {
+    /// <summary>
+    /// Represents the format of an audio sample.
+    /// </summary>
+    public class SampleFormat {
+        /// <summary>
+        /// The audio bit depth, i.e. how many bits are used per a single
+        /// channel in a sample.
+        /// </summary>
+        public AudioBitDepth bitDepth;
```

Comment:
Can you please capitalize the public fields?

Reply:
Wouldn't that be a violation of C#'s naming convention, though? IIRC, fields should always be `camelCase`, except when constant (`static readonly`), which would be `PascalCase`

Reply:
From https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions "When naming public members of types, such as fields, properties, events, methods, and local functions, use pascal casing." 

It is more important that its `public` than that it is a field.